### PR TITLE
fix: Examination title in proctor console

### PR DESCRIPTION
**Pull request by yaraslau-kavaliou**
https://oat-sa.atlassian.net/browse/OATSD-1664

### CODE REVIEW

**Comment by poyuki** [comment]
@tikhanovichA PR litle bit updated

File: model/execution/DeliveryExecutionList.php [hunk 1]
```diff
@@ -115,7 +115,13 @@ private function createExecution($cachedData, $extraFields): array
             'id' => $cachedData[DeliveryMonitoringService::DELIVERY_EXECUTION_ID],
             'delivery' => array(
                 'uri' => $cachedData[DeliveryMonitoringService::DELIVERY_ID],
-                'label' => $this->sanitizeUserInput($cachedData[DeliveryMonitoringService::DELIVERY_NAME]),
+                'label' => $this->sanitizeUserInput((
+                    array_key_exists(DeliveryMonitoringService::DELIVERY_BATTERY_NAME, $cachedData) &&
```
Comment (tikhanovichA) [inline model/execution/DeliveryExecutionList.php:119]: I think we should delete mentions of batteries in the taoProctoring extension

File: model/monitorCache/DeliveryMonitoringService.php [hunk 2]
```diff
@@ -70,6 +70,7 @@ interface DeliveryMonitoringService extends DeliveryExecutionDelete
     const DELIVERY_ID = 'delivery_id';
     const DELIVERY_NAME = 'delivery_name';
     const CONNECTIVITY = 'last_connect';
+    const DELIVERY_BATTERY_NAME = 'delivery_battery_name';
```
Comment (tikhanovichA) [inline model/monitorCache/DeliveryMonitoringService.php:73]: the same. Should be implemented on the nccer side

**Review by tikhanovichA — CHANGES_REQUESTED** [review]

File: model/execution/DeliveryExecutionList.php (hunk 1 above)
Comment (poyuki) [reply model/execution/DeliveryExecutionList.php:119]: is battery extension specific for nccer?  we don't need override label by battery name in common?

Comment (tikhanovichA) [reply model/execution/DeliveryExecutionList.php:119]: yes, taoProctoring is not linked with batteries. I think based on the second PR you can just revert those changes

File: model/datatable/DeliveriesMonitorDatatable.php [hunk 3]
```diff
@@ -135,6 +136,6 @@ public function jsonSerialize()
      */
     private function getDeliveryHelperService()
     {
-        return $this->getServiceLocator()->get(DeliveryExecutionList::class);
+        return $this->getServiceLocator()->get(DeliveryExecutionListInterface::SERVICE_ID);
```
Comment (wazelin) [inline model/datatable/DeliveriesMonitorDatatable.php:139]: `DeliveryExecutionList::class` or `DeliveryExecutionListInterface::class` should still work just fine.
Let's try minimizing the deprecated `SERVICE_ID` usage.

Comment (poyuki) [reply model/datatable/DeliveriesMonitorDatatable.php:139]: you are right, i missed that magic https://github.com/oat-sa/generis/blob/master/common/oatbox/service/ServiceManager.php#L97

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by tikhanovichA — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are me

**Pull request merged**